src/core/Clock.js

```diff
@@ -68,6 +64,14 @@ THREE.Clock.prototype = {
 
 		return diff;
 
+	},
+
+	getNow: function () {
```

Comment:
The problem of this is that now people will expect this method as part of the API...
I would maybe name it `_getNow` so they can see that is somewhat internal.